features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1385,3 +1385,26 @@ REFERENCING NEW AS NEW FOR EACH ROW MODE DB2SQL
 
 CREATE INDEX IDX_AT_SI_ECI ON IDN_OIDC_SCOPE_CLAIM_MAPPING(SCOPE_ID, EXTERNAL_CLAIM_ID)
 /
+
+CREATE TABLE IDN_TEMPLATE_MGT(
+    TEMPLATE_ID INTEGER NOT NULL,
+    TENANT_ID INTEGER NOT NULL,
+    NAME VARCHAR(255) NOT NULL,
+    DESCRIPTION VARCHAR(1023),
+    TEMPLATE_SCRIPT BLOB NOT NULL,
+    PRIMARY KEY (TEMPLATE_ID),
+    UNIQUE(TENANT_ID, NAME)
+)
+/
+CREATE SEQUENCE TEMPLATE_ID START WITH 1 INCREMENT BY 1 NOCACHE
+/
+
+CREATE TRIGGER IDN_TEMPLATE_MGT NO CASCADE
```

Comment:
Shall we rename the trigger to match others?

Reply:
ok